pkg/controller/types/networkpolicy.go

```diff
@@ -119,4 +119,11 @@ type NetworkPolicy struct {
 	Rules []networkpolicy.NetworkPolicyRule
 	// AppliedToGroups is a list of names of AppliedToGroups to which this policy applies.
 	AppliedToGroups []string
+	// List of directions that the NetworkPolicy relates to.
+	// Valid options are "In", "Out", or "In,Out".
```

Comment:
if the PolicyTypes is not specified then it defaults to ingress .. unless there are egress rules .. then it applies to ingress and egress

Reply:
>  If no policyTypes are specified on a NetworkPolicy then by default Ingress will always be set and Egress will be set if the NetworkPolicy has any egress rules.


 from the NetworkPolicy docs